README.md

```diff
@@ -12,7 +14,7 @@ Helm's [documentation](https://helm.sh/docs) to get started.
 
 Once Helm has been set up correctly, add the repo as follows:
 
-    helm repo add uptime-kuma https://dirsigler.github.io/uptime-kuma-helm
+    helm repo add uptime-kuma https://chrisbsmith.github.io/uptime-kuma-helm
```

Comment:
As above:
This should not change, as the source still remains here in this repository.